include/roboteam_ai/stp/StpInfo.h

```diff
@@ -105,6 +117,9 @@ struct StpInfo {
     int getDribblerSpeed() const { return dribblerSpeed; }
     void setDribblerSpeed(int dribblerSpeed) { this->dribblerSpeed = dribblerSpeed; }
 
+    KickChipType getKickChipType() const { return kickChipType; }
+    void setKickChipType(KickChipType kickChipType) { StpInfo::kickChipType = kickChipType; }
```

Comment:
Not that familiar with cpp, but doesn't this imply that the field is static?

Reply:
o woops, autmatically generated setters aren't great